src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,25 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
-    public final String value;
+    public static final String EXAMPLE = "123, some street, some unit, some postal code";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = 
+    		"Person addresses must consist of four strings seperated by commas";
+    public static final String FIELD_SEPARATOR = ", ";
+    
+    private static final int BLOCK_INDEX = 0;
```

Comment:
it is good that you are using constants to eliminate magic numbers :+1: 

however, please check how you should name the variables?  please recollect the discussion on this during the tutorial